### PR TITLE
DFE-839 - Google analytics file download event release page

### PR DESCRIPTION
**Pull request by HiveIT-Lewis**

### CODE REVIEW

File: src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx [hunk 1]
```diff
@@ -145,6 +145,10 @@ class PublicationReleasePage extends Component<Props> {
                       <Link
                         to={`${baseUrl.data}/api/download/${path}`}
                         className="govuk-link"
+                        analytics={{
+                          category: 'Downloads',
+                          action: `Release page ${name} file downloaded`,
```
Comment (jamescutts) [inline src/explore-education-statistics-frontend/src/modules/find-statistics/PublicationReleasePage.tsx]: I think we'll also want to be tracking the url of the file that is downloaded

File: src/explore-education-statistics-frontend/src/components/Link.tsx [hunk 2]
```diff
@@ -31,7 +31,11 @@ const Link = ({
 }: LinkProps) => {
   const handleAnalytics = () => {
     if (analytics) {
-      logEvent(analytics.category, analytics.action, window.location.pathname);
+      logEvent(
+        analytics.category,
+        analytics.action,
+        analytics.label ? analytics.label : window.location.pathname,
```
Comment (jamescutts) [inline src/explore-education-statistics-frontend/src/components/Link.tsx]: Isn't this the location of the browser window not the link?

Comment (HiveIT-Lewis) [reply src/explore-education-statistics-frontend/src/components/Link.tsx]: The label is the link. If there is no link it replaces the link with the location of the browser window

Comment (duncan-at-hiveit) [reply src/explore-education-statistics-frontend/src/components/Link.tsx]: FYI I think you can short-hand this as `analytics.label || window.location.pathname`, for next time

File: src/explore-education-statistics-frontend/src/components/Page.tsx [hunk 3]
```diff
@@ -19,7 +19,7 @@ type Props = {
 
 const Page = ({
   title,
-  caption = 'Find, download and explore official Department for Education (DfE) statistics and data in England.',
+  caption = '',
```
Comment (duncan-at-hiveit) [inline src/explore-education-statistics-frontend/src/components/Page.tsx]: I think it the caption has been blanked out deliberately here?

Comment (HiveIT-Lewis) [reply src/explore-education-statistics-frontend/src/components/Page.tsx]: Yes it's been done deliberately

**Pull request merged**